routing/control_tower.go

```diff
@@ -109,9 +115,10 @@ type controlTower struct {
 // NewControlTower creates a new instance of the controlTower.
 func NewControlTower(db *channeldb.PaymentControl) ControlTower {
 	return &controlTower{
-		db:          db,
-		subscribers: make(map[lntypes.Hash][]*ControlTowerSubscriber),
-		paymentsMtx: multimutex.NewHashMutex(),
+		db:                     db,
+		subscribersAllPayments: make([]*ControlTowerSubscriber, 0),
```

Comment:
Slices generally don't need to be initialized, unless they should be pre-sized to a specific length to avoid re-sizing at runtime. So I think this line can just be removed.

Reply:
Done